pkg/helm/build.go

```diff
@@ -4,16 +4,22 @@ import (
 	"fmt"
 )
 
-const helmClientVersion = "v2.12.3"
-const dockerfileLines = `RUN apt-get update && \
+const helmClientVersion string = "v2.12.3"
```

Comment:
Now that we can autodownload the right version of helm. Do you recommend that we start keeping this current with latest patch release of helm? (not in this PR but in general)

Reply:
Yes!  Might as well update here 👍

Reply:
Updated.